mmseg/datasets/pipelines/transforms.py

```diff
@@ -1040,3 +1042,261 @@ def __repr__(self):
         repr_str += f'fill_in={self.fill_in}, '
         repr_str += f'seg_fill_in={self.seg_fill_in})'
         return repr_str
+
+
+@PIPELINES.register_module()
+class RandomMosaic(object):
+    """Mosaic augmentation. Given 4 images, mosaic transform combines them into
+    one output image. The output image is composed of the parts from each sub-
+    image.
+
+    .. code:: text
+                        mosaic transform
+                           center_x
+                +------------------------------+
+                |       pad        |  pad      |
+                |      +-----------+           |
+                |      |           |           |
+                |      |  image1   |--------+  |
+                |      |           |        |  |
+                |      |           | image2 |  |
+     center_y   |----+-------------+-----------|
+                |    |   cropped   |           |
+                |pad |   image3    |  image4   |
+                |    |             |           |
+                +----|-------------+-----------+
+                     |             |
+                     +-------------+
+     The mosaic transform steps are as follows:
+         1. Choose the mosaic center as the intersections of 4 images
+         2. Get the left top image according to the index, and randomly
+            sample another 3 images from the custom dataset.
+         3. Sub image will be cropped if image is larger than mosaic patch
```

Comment:
the code block is not be rendered
![image](https://user-images.githubusercontent.com/76149310/148538881-70cea5b4-d898-4e0e-a4f1-18769d0451ac.png)

Reply:
Sorry, I don't understand your comment.
Could you explain a bit more?

Reply:
Here is a code block in docstring https://github.com/open-mmlab/mmcv/blob/d30e37d4d57425843bbff484e7e7a1e41230381f/mmcv/ops/deform_conv.py#L288-L292
and it will be rendered like 
![image](https://user-images.githubusercontent.com/76149310/148552742-756779e5-e14a-492c-9713-70bc561f9699.png)
https://mmcv.readthedocs.io/en/latest/api.html#mmcv.ops.DeformConv2d.forward

I just check the rendering for `RandomMosaic` 
https://mmsegmentation--1093.org.readthedocs.build/en/1093/api.html#mmseg.datasets.pipelines.RandomMosaic
but it might meet some problems when rendering.

Reply:
I found if add blank line after `.. code:: text`, everything will be ok.
```suggestion
    .. code:: text
   
                        mosaic transform
                           center_x
                +------------------------------+
                |       pad        |  pad      |
                |      +-----------+           |
                |      |           |           |
                |      |  image1   |--------+  |
                |      |           |        |  |
                |      |           | image2 |  |
     center_y   |----+-------------+-----------|
                |    |   cropped   |           |
                |pad |   image3    |  image4   |
                |    |             |           |
                +----|-------------+-----------+
                     |             |
                     +-------------+
     
     The mosaic transform steps are as follows:
         1. Choose the mosaic center as the intersections of 4 images
         2. Get the left top image according to the index, and randomly
            sample another 3 images from the custom dataset.
         3. Sub image will be cropped if image is larger than mosaic patch
```

Reply:
I understand it now. Thanks for fixing.